store/store_test.go

```diff
@@ -382,11 +392,14 @@ func TestLoadBaseMeta(t *testing.T) {
 	require.NoError(t, err)
 	bs := NewBlockStore(memdb.NewDB())
 
+	lastCommit := &types.Commit{BlockID: types.EmptyBlockID()}
+
 	for h := int64(1); h <= 10; h++ {
-		block := makeBlock(h, state, new(types.Commit))
+		block := makeBlock(h, state, lastCommit)
 		partSet := block.MakePartSet(2)
-		seenCommit := makeTestCommit(h, tmtime.Now())
-		bs.SaveBlock(block, partSet, seenCommit)
+		lastCommit := makeTestCommit(block, h, tmtime.Now())
+		block.Header.LastBlockID = lastCommit.BlockID
```

Comment:
Maybe move this line into `makeBlock`?

Reply:
This would simplify some code here and below

Reply:
It would make things simpler here, but we'd have to add logic to use an empty block id when the commit is nil, and lose some explicitness. I did however remove an unnecessary line https://github.com/lazyledger/lazyledger-core/pull/312/commits/17055a67fc101d161fde0828729b4b8972592ac3